src/core/title.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * This module handles the creation of the h1#title of a spec and
+ * makes sure the <title> always matches the h1.
+ *
+ * If no h1#title is included, then the <title> becomes the h1#title.
+ *
+ * When a h1#title is included, it always takes precedence over the
+ * <title> of a spec. An error will be displayed in case of
+ * any mismatch.
+ *
+ */
+
+import { getIntlData, norm, showInlineError } from "./utils.js";
+import { hyperHTML } from "./import-maps.js";
+export const name = "core/title";
+
+const localizationStrings = {
+  en: {
+    default_title: "No Title",
+  },
+  de: {
+    default_title: "Kein Titel",
+  },
+};
+
+const l10n = getIntlData(localizationStrings);
+
+export function run(conf) {
+  /** @type {HTMLElement} */
+  const h1Elem =
+    document.querySelector("h1#title") || document.createElement("h1");
```

Comment:
```suggestion
    document.querySelector("h1#title") || hyperHTML`<h1 id="title">`;
```

This can skip assigning id below.

Reply:
ooh nice. thanks!